.github/workflows/dependencies.yml

```diff
@@ -1,27 +1,29 @@
-name: Bumper
+name: Update this project's version where it servers as a dependency
 on:
   push:
     branches:
       - master
   workflow_dispatch:
 
 jobs:
-  bumpProjects:
+  update_version:
+    # Pushes new refs to interested external repositories, so they can do early testing against them.
     runs-on: ubuntu-latest
+    name: Update libp2p's version for ${{ matrix.target.repo }} from ${{ matrix.target.source_ref }}
     strategy:
       fail-fast: false
       matrix:
         target: [
-          { repo: status-im/nimbus-eth2, branch: unstable },
-          { repo: waku-org/nwaku, branch: master },
-          { repo: codex-storage/nim-codex, branch: master }
+          { repo: status-im/nimbus-eth2, source_ref: master },
```

Comment:
this should be unstable

Reply:
Isn't `unstable` deprecated? I asked on discord but I got no answer.

Reply:
nimbus unstable isn't deprecated afaik

Reply:
I'm also not sure there's a need to rename "branch"

Reply:
Ah! Now I understand, I got confused with the names. Thanks for pointing that out.